app/assets/stylesheets/layouts/starry.scss

```diff
@@ -1,85 +1,217 @@
 @import 'variables';
+
+/* - header */
+$bg_color_header_top: transparent; // top header strip for logo
+$bg_color_header_middle: #05050F; // middle header strip with user items
+$bg_color_header_bottom: #484357; // bottom header strip with navigation
+
 #header {
-  background-color: #484357;
+  background-color: $bg_color_header_bottom;
   background-image: url(image-path("layouts/starry.jpg"));
   background-position: right top;
 }
-#logo, #nav-bottom { background-color: transparent !important; }
-#user-info, #nav-top { background-color: #05050F; background-color: rgba(5, 5, 15, 0.9); }
+#logo { background-color: $bg_color_header_top !important; }
+#user-info, #nav-top { background-color: $bg_color_header_middle; background-color: rgba($bg_color_header_middle, 0.9); }
 
-body { background-color: #6C697C; }
-#character-icon-selector, .time-loaded { color: #DADADA; }
+/* - links */
+$font_color_link: #666284; // link
+$font_color_link_visited: #9C9AA4;
 
-th, .content-header { background-color: #13132F; }
-.sub { color: #F3F3F3; }
-.sub, .post-navheader, .flash.breadcrumbs { background-color: #182152; }
-.form-table-ender { background-color: #13132F; color: #F3F3F3; }
-.selector-popup, .single-description, .subber { background-color: #484357; color: #DADADA; }
-th, .single-description, .content-header {
-  a { color: #B5AFCB; }
-  a:visited { color: #9C9AA4; }
+a { color: $font_color_link; }
+a:visited { color: #484357; } //TODO: inconsistency between this a:visited and others, ???? this was previous at the bottom of the documment, was it overriding?
```

Comment:
Things only override things of same of less specificity. So here, since it's `a:visited`, it has less specificity than any of the other `… a:visited`s in the document (`.subber, .flash.breadcrumbs { a:visited { … } }`, `.post-screenname a:visited`, `.post-character, .post-author { a:visited { … } }`, `.pagination a:visited`), and so its order chronologically has no effect (all the other styles take precedence).

You should be able to extract this into the `$font_color_link_visited` thing like normal.